src/test/java/junit/tests/framework/AssertTest.java

```diff
@@ -82,15 +82,15 @@ public void testAssertStringNotEqualsNull() {
             assertEquals("foo", null);
             fail();
         } catch (ComparisonFailure e) {
-            e.getMessage(); // why no assertion?
+            String unused = e.getMessage(); // why no assertion?
```

Comment:
I suggest we use `assertNotNull()` here and below.

Reply:
Sure, done.